perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp

```diff
@@ -52,51 +52,57 @@ BicycleTracker::BicycleTracker(
 {
   object_ = object;
 
-  // initialize params
-  float q_stddev_x = 0.6;                                     // [m/s]
-  float q_stddev_y = 0.6;                                     // [m/s]
-  float q_stddev_yaw = tier4_autoware_utils::deg2rad(10);     // [rad/s]
-  float q_stddev_vx = tier4_autoware_utils::kmph2mps(10);     // [m/(s*s)]
-  float q_stddev_slip = tier4_autoware_utils::deg2rad(15);    // [rad/(s*s)]
-  float r_stddev_x = 0.6;                                     // [m]
-  float r_stddev_y = 0.4;                                     // [m]
-  float r_stddev_yaw = tier4_autoware_utils::deg2rad(30);     // [rad]
-  float p0_stddev_x = 0.8;                                    // [m/s]
-  float p0_stddev_y = 0.5;                                    // [m/s]
-  float p0_stddev_yaw = tier4_autoware_utils::deg2rad(1000);  // [rad/s]
-  float p0_stddev_vx = tier4_autoware_utils::kmph2mps(1000);  // [m/(s*s)]
-  float p0_stddev_slip = tier4_autoware_utils::deg2rad(10);   // [rad/(s*s)]
-  ekf_params_.q_cov_x = std::pow(q_stddev_x, 2.0);
-  ekf_params_.q_cov_y = std::pow(q_stddev_y, 2.0);
-  ekf_params_.q_cov_yaw = std::pow(q_stddev_yaw, 2.0);
-  ekf_params_.q_cov_vx = std::pow(q_stddev_vx, 2.0);
-  ekf_params_.q_cov_slip = std::pow(q_stddev_slip, 2.0);
+  // Initialize parameters
+  // measurement noise covariance: detector uncertainty + ego vehicle motion uncertainty
+  float r_stddev_x = 0.5;                                  // in vehicle coordinate [m]
+  float r_stddev_y = 0.4;                                  // in vehicle coordinate [m]
+  float r_stddev_yaw = tier4_autoware_utils::deg2rad(30);  // [rad]
   ekf_params_.r_cov_x = std::pow(r_stddev_x, 2.0);
   ekf_params_.r_cov_y = std::pow(r_stddev_y, 2.0);
   ekf_params_.r_cov_yaw = std::pow(r_stddev_yaw, 2.0);
+  // initial state covariance
+  float p0_stddev_x = 0.8;                                     // [m]
+  float p0_stddev_y = 0.5;                                     // [m]
+  float p0_stddev_yaw = tier4_autoware_utils::deg2rad(25);     // in map coordinate [rad]
+  float p0_stddev_vel = tier4_autoware_utils::kmph2mps(1000);  // in object coordinate [m/s]
+  float p0_stddev_slip = tier4_autoware_utils::deg2rad(5);     // in object coordinate [rad/s]
   ekf_params_.p0_cov_x = std::pow(p0_stddev_x, 2.0);
   ekf_params_.p0_cov_y = std::pow(p0_stddev_y, 2.0);
   ekf_params_.p0_cov_yaw = std::pow(p0_stddev_yaw, 2.0);
-  ekf_params_.p0_cov_vx = std::pow(p0_stddev_vx, 2.0);
+  ekf_params_.p0_cov_vel = std::pow(p0_stddev_vel, 2.0);
   ekf_params_.p0_cov_slip = std::pow(p0_stddev_slip, 2.0);
-  max_vx_ = tier4_autoware_utils::kmph2mps(100);  // [m/s]
-  max_slip_ = tier4_autoware_utils::deg2rad(30);  // [rad/s]
-
-  // initialize X matrix
+  // process noise covariance
+  ekf_params_.q_stddev_acc_long = 9.8 * 0.35;  // [m/(s*s)] uncertain longitudinal acceleration
```

Comment:
Just a question. How did you tuned these parameter?

Reply:
To be honest, these parameters are decided by experience (replay simulation and visually check), which is not a scientific method.
I would like to see rather objective solution such as data-driven method to determine those parameters.